cmd/main.go

```diff
@@ -167,14 +167,19 @@ func runBuild(ctx context.Context, args []string) error {
 		log.Fatalf("[FATAL] %v", err)
 	}
 
+	// done if we're skipping the build
+	if builder.SkipBuild {
+		return nil
+	}
```

Comment:
This is after the build though?

Reply:
Yes. or else `set cap` and `run after build` is broken, because there is no binary at all. `run after build` is broken because these envs can be empty. Simply start a golang docker and build caddy and there is no running after build.